lib/phlex/sgml/elements.rb

```diff
@@ -23,6 +56,7 @@ def #{method_name}(**attributes)
 					if block_given # with content block
 						buffer << "<#{tag}"
 						begin
+							#{COMMA_SEPARATED_TOKENS[method_name]}
```

Comment:
This outputs nothing for most elements and otherwise outputs the code to pre-process one or two values if they exist before caching.